deploy/operator.yaml

```diff
@@ -233,10 +233,29 @@ rules:
   verbs:
   - delete
   - update
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - list
+  - get
+  - watch
+  - create
+  - delete
+  - update
 - apiGroups:
   - ""
   resources:
   - pods
+  verbs:
+  - get
+  - list
+  - watch
+  - delete
```

Comment:
delete but not create?

Reply:
Yes it is kind of strange but if someone deletes the SA, I have to delete pods, so that their respective daemonsets/deployments re-create them. https://github.com/kubevirt/hostpath-provisioner-operator/pull/175/files#diff-e8884cf889bfc17772c62395d1b32d34a2e998e98c65a763bc6b839f39b8b17dR116. The reason being the SA secret has changed, and thus the pods need to be re-created.

Reply:
Still don’t understand.  Are these pods created by a deployment?

Reply:
Why not delete the deployment?

Reply:
Good point, its both deployment and daemonset, but I can just delete those instead of the pods, that would trigger the operator to recreate them and thus recreate the pods.